src/common.rs

```diff
@@ -467,4 +467,6 @@ pub enum SymbolFlags<Section> {
         /// `Number` field in the auxiliary symbol for the section.
         associative_section: Option<Section>,
     },
+    /// XCOFF symbol flags.
+    Xcoff { c_info: u8 },
```

Comment:
This needs to be similar to other file formats, where we put the raw XCOFF symbol field here:
```suggestion
    Xcoff { n_type: u16 },
```

Also please change `read::XcoffSymbol::flags` to return `SymbolFlags::Xcoff`.